### PR TITLE
Webnam 69

### PR DESCRIPTION
**Pull request by bsuttor**
https://support.imio.be/browse/WEBNAM-69

### CODE REVIEW

File: cpskin/workflow/permissions.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+from AccessControl.SecurityInfo import ModuleSecurityInfo
+from Products.CMFCore.permissions import setDefaultRoles
+
+
+security = ModuleSecurityInfo('cpskin.locales')
```
Comment (laulaz) [inline cpskin/workflow/permissions.py]: Should be 'cpskin.workflow' instead of locales, isn't it ?

Comment (bsuttor) [reply cpskin/workflow/permissions.py]: Indeed

File: cpskin/workflow/upgradehandlers.py [hunk 2]
```diff
@@ -5,6 +5,10 @@
 from zope.ramcache.interfaces.ram import IRAMCache
 
 
+def add_private_view(context):
+    context.runAllImportStepsFromProfile('profile-cpskin.workflow:default')
```
Comment (laulaz) [inline cpskin/workflow/upgradehandlers.py:9]: Not too dangerous to reinstall everything ?
It scares me sometimes 😄

Comment (bsuttor) [reply cpskin/workflow/upgradehandlers.py:9]: It's just workflow package, usually nobody change portal workflow ;-)

**Review by laulaz — APPROVED** [review]

**Pull request merged**